api-gateway/default.conf

```diff
@@ -80,14 +80,22 @@ server {
         proxy_pass http://history-service:8005/history/;
     }
 
-    location / {
```

Comment:
Should we also create an exact match directive to `/` with a hello world

Reply:
Why